src/wp-admin/includes/class-wp-automatic-updater.php

```diff
@@ -1011,26 +1011,99 @@ protected function send_plugin_theme_email( $type, $successful_updates, $failed_
 			$body[] = "\n";
 			$body[] = __( 'Please check your site now. It’s possible that everything is working. If there are updates available, you should update.' );
 			$body[] = "\n";
-
+            
+            // Check for failed themes and plugins listing with failed_update_plugins_themes option_name
+			$failed_update_plugins_themes = (array) get_site_option( 'failed_update_plugins_themes', array() );
+			
 			// List failed plugin updates.
 			if ( ! empty( $failed_updates['plugin'] ) ) {
 				$body[] = __( 'These plugins failed to update:' );
 
 				foreach ( $failed_updates['plugin'] as $item ) {
 					$body[] = "- {$item->name}";
+                    
+                    /*
+					 * Failed Plugin list to store into the database
+					 */ 
+					$failed_plugin = array( 'name' => $item->name, 'failure_time' => time(), 'file_name' => $item->item->plugin );
+					if( isset( $failed_update_plugins_themes['plugin'] ) && !empty( $failed_update_plugins_themes['plugin'] ) ){
```

Comment:
Hi,

Thanks for the patch. There are some coding standards issues here. There should be a space between "if" and the opening parenthesis, same after "!" operator or before the opening bracket at eh end of this line.